src/System.Data.SqlClient/pkg/System.Data.SqlClient.pkgproj

```diff
@@ -13,6 +13,11 @@
     <HarvestIncludePaths Include="ref/netstandard1.3">
       <SupportedFramework>net46;netcoreapp1.0</SupportedFramework>
     </HarvestIncludePaths>
+    <HarvestIncludePaths Include="ref/netstandard2.0">
+      <SupportedFramework>netcoreapp2.0;uap10.0.16299;$(AllXamarinFrameworks)</SupportedFramework>
+    </HarvestIncludePaths>
+    <HarvestIncludePaths Include="lib/netstandard1.2;lib/netstandard1.3;lib/netstandard2.0" />
```

Comment:
Style: we try to match up the TFMs, so rather than `lib/netstandard1.2;lib/netstandard1.3;lib/netstandard2.0` here can you do `ref/netstandard1.2;lib/netstandard1.2` `ref/netstandard1.3;lib/netstandard1.3` `ref/netstandard2.0;lib/netstandard2.0` above?